contract_api/registry.py

```diff
@@ -431,3 +432,24 @@ def curate_service(self, org_id, service_id, curated):
             service_repo.curate_service(org_id, service_id, 0)
         else:
             Exception("Invalid curation flag")
+
+    def get_service_media(self,org_id,service_id):
```

Comment:
add logging. if the exception is raised we should be getting `org_id` or `service_id` in logs.

Reply:
logging is added